tests/common.ts

```diff
@@ -13,6 +13,23 @@ export function testWithClient(
   });
 }
 
+export function compareVersion(
```

Comment:
I think you can use the semver module to compare versions
https://deno.land/x/semver@v1.4.0